scripts/parse_prophecy.py

```diff
@@ -15,33 +15,79 @@
 
 from sample_metadata.parser.generic_metadata_parser import (
     GenericMetadataParser,
-    GroupedRow,
+    SingleRow,
     run_as_sync,
 )
 
 logger = logging.getLogger(__file__)
 logger.addHandler(logging.StreamHandler())
 logger.setLevel(logging.INFO)
 
-SEQUENCE_META_MAP = {
+PROJECT = 'prophecy-test'
+
+COLUMN_MAP = {
     'Application': 'platform',
     'External ID': 'external_id',
     'Sample Concentration (ng/ul)': 'concentration',
     'Volume (uL)': 'volume',
-}
-
-SAMPLE_META_MAP = {
     'Sex': 'sex',
 }
 
-PROJECT = 'prophecy-test'
+
+class Columns:
+    """Column keys for Prophecy manifest"""
+
+    EXTERNAL_ID = 'External ID'
+    PLATFORM = 'Application'
+    CONCENTRATION = 'Sample Concentration (ng/ul)'
+    VOLUME = 'Volume (uL)'
+    SEX = 'Sex'
+    # SAMPLE_NAME != External Sample ID
+    # Follows XXXXXX_FLUIDX convention.
+    SAMPLE_NAME = 'Sample/Name'
+
+    @staticmethod
+    def participant_meta_map():
+        """Participant meta map"""
+        return {}
+
+    @staticmethod
+    def sequence_meta_map():
+        """Columns that will be put into sequence.meta"""
+        fields = [
+            Columns.PLATFORM,
+            Columns.CONCENTRATION,
+            Columns.VOLUME,
+        ]
+        return {k: COLUMN_MAP[k] for k in fields}
+
+    @staticmethod
+    def sample_meta_map():
+        """Columns that will be put into sample.meta"""
+        fields = [Columns.SEX]
```

Comment:
Shouldn't Sex go on the participant?

Reply:
Yes! Will fix.

Reply:
Actually, it's redundant. Sex is parsed anyway on the participant.

Reply:
Will scrap.